.github/workflows/integration-test-opentofu.yml

```diff
@@ -0,0 +1,109 @@
+name: Integration Test - OpenTofu
+
+# Idea: 
+# 1. Build the provider from the latest pull
+# 2. Setup OpenTofu
+# 3. Execute the integration test script
+# 4. All changes must apply
+# 5. Destroy the infrastructure
+
+
+on:
+  workflow_dispatch:
+
+env:
+  PATH_TO_TFSCRIPT: 'regression-test'
+  DEV_OVERRIDE: 'assets/provider-config-ghaction.txt'
+  TEMP_PLAN_OUTPUT : 'plan-output.txt'
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - name: Check out Git repository
+        id: checkout_repo
+        uses: actions/checkout@v4
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: 'go.mod'
+          cache: false
+      - run: go mod download
+      - run: go mod tidy
+      - run: go build -v .
+      - name: Run linters
+        uses: golangci/golangci-lint-action@v8
+        with:
+          version: latest
+          skip-cache: true
+
+  test:
+    name: Integration Test for OpenTofu
+    needs: build
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out Git repository
+      id: checkout_repo
+      uses: actions/checkout@v4
+
+    - name: Setup Go
+      uses: actions/setup-go@v5
+      with:
+        go-version-file: 'go.mod'
+        cache: false
+
+    - name: Setup OpenTofu
+      id : setup_opentofu
+      uses: opentofu/setup-opentofu@v1
+      with:
+        tofu_wrapper: false
+        tofu_version: latest
+
+    - name: Install local Terraform provider
+      id: build_provider
+      shell: bash
+      run: |
+        go mod tidy && make install
+
+    - name: Remove init data and place override
+      id: set_to_override
+      shell: bash
+      run: |
+        rm -rf .terraform && rm -rf .terraform.lock.hcl && cp ${{ env.DEV_OVERRIDE }} ~/.terraformrc
+
+    - name: Tofu Plan
+      run: |
+        export BTP_USERNAME=${{ secrets.BTP_USERNAME }}
+        export BTP_PASSWORD=${{ secrets.BTP_PASSWORD }}
+        tofu -chdir=${{ env.PATH_TO_TFSCRIPT }} plan -var "globalaccount=${{ secrets.GLOBALACCOUNT }}" -out=${{ env.TEMP_PLAN_OUTPUT }} -no-color
+
+
+    - name: Tofu Apply with local provider
+      id: tofu_apply_local
+      shell: bash
+      run: |
+        export BTP_USERNAME=${{ secrets.BTP_USERNAME }}
+        export BTP_PASSWORD=${{ secrets.BTP_PASSWORD }}
+        tofu -chdir=${{ env.PATH_TO_TFSCRIPT }} apply -var globalaccount=${{ secrets.GLOBALACCOUNT }} -auto-approve -no-color
+    
+    - name: tofu Destroy with local provider
+      if: always()
+      id: tofu_destroy
+      shell: bash
+      run: |
+        export BTP_USERNAME=${{ secrets.BTP_USERNAME }}
+        export BTP_PASSWORD=${{ secrets.BTP_PASSWORD }}
+        tofu -chdir=${{ env.PATH_TO_TFSCRIPT }} destroy -var globalaccount='${{ secrets.GLOBALACCOUNT }}' -auto-approve -no-color
+
+    - name: Cleanup plan output
```

Comment:
This step can be removed as the VM will anyway be removed, so no side effects to be expected